package.json

```diff
@@ -36,7 +36,8 @@
     "adapter:docs:generate": "npx tsx ./build/generateAdapterPropDocs.ts",
     "watch:vite": "vite build --watch --mode development",
     "watch:translations": "node ./build/translationWatcher.js",
-    "dev": "node ./build/prompt.js && npm run i18n:generate && npm-run-all --parallel watch:vite watch:translations",
+    "dev": "node ./build/prompt.js && npm run i18n:generate && npm-run-all --parallel watch:vite watch:translations ladle",
+    "dev:setup": "npm link ../gws-flows/node_modules/react && (cd ../gws-flows && yarn link -r ../embedded-react-sdk)",
```

Comment:
You can run this after you get latest on GWSFlows and SDK to get back to a working state